conf/modules.config

```diff
@@ -134,10 +134,18 @@ process {
             ]
         }
 
-    withName: 'OPENMS_DECOYDATABASE|OPENMS_THERMORAWFILEPARSER|OPENMS_PEAKPICKERHIRES|OPENMS_PEPTIDEINDEXER|OPENMS_FALSEDISCOVERYRATE|OPENMS_IDMERGER|OPENMS_MAPRTTRANSFORMER.*ML|OPENMS_FEATURE.*|OPENMS_IDCONFLICTRESOLVER|OPENMS_MZTABEXPORTER' {
+    withName: 'OPENMS_DECOYDATABASE|OPENMS_THERMORAWFILEPARSER|OPENMS_PEAKPICKERHIRES|OPENMS_PEPTIDEINDEXER|OPENMS_FALSEDISCOVERYRATE|OPENMS_IDMERGER|OPENMS_MAPRTTRANSFORMER.*ML|OPENMS_FEATURE.*|OPENMS_IDCONFLICTRESOLVER' {
             publishDir       = [ enabled: false ]
         }
 
+    withName: 'OPENMS_MZTABEXPORTER_QUANT' {
```

Comment:
afaik definitions of `OPENMS_MZTABEXPORTER` are inherited to `OPENMS_MZTABEXPORTER_QUANT` because `include { OPENMS_MZTABEXPORTER as OPENMS_MZTABEXPORTER_QUANT }`. I typically try to avoid specifying the base name (i.e. here `OPENMS_MZTABEXPORTER`) but rather rename all processes (i.e. use `include as`) so that those confusions do not happen. Just a comment. In this case here all should go fine.